src/wkwebview/mod.rs

```diff
@@ -806,10 +808,19 @@ impl InnerWebView {
             run_file_upload_panel as extern "C" fn(&Object, Sel, id, id, id, id),
           );
 
-          ctl.add_method(
-            sel!(webView:requestMediaCapturePermissionForOrigin:initiatedByFrame:type:decisionHandler:),
-            request_media_capture_permission as extern "C" fn(&Object, Sel, id, id, id, id, id),
-          );
+          #[cfg(target_os = "macos")]
+          match operating_system_version() {
+            (14, 0, _) => {
+              // Do not suppress media request on 14.0:
+              // https://github.com/tauri-apps/wry/issues/1101
+            }
```

Comment:
```suggestion
            (14, 0, _) | (14, 1, _) => {
              // Do not suppress media request on 14.0 & 14.1:
              // https://github.com/tauri-apps/wry/issues/1101
            }
```
https://github.com/tauri-apps/wry/issues/1195#issuecomment-2009402427